src/DIRAC/FrameworkSystem/Service/TokenManagerHandler.py

```diff
@@ -110,41 +114,52 @@ def export_getUsersTokensInfo(self, users):
                     else:
                         for tokenDict in result["Value"]:
                             if tokenDict not in tokensInfo:
+                                # The database does not contain a username,
+                                # as it is a unique user ID exclusively for DIRAC
+                                # and is not associated with a token.
                                 tokenDict["username"] = user
                                 tokensInfo.append(tokenDict)
         return S_OK(tokensInfo)
 
-    auth_uploadToken = ["authenticated"]
+    auth_updateToken = ["authenticated"]
```

Comment:
I don't think it's a good idea to hardcode these `auth_` properties. I think it's better to add them to the ConfigTemplate.cfg file.

Reply:
Auth properties in `ConfigTemplate.cfg` will rewrite this (in `auth_ `), no?

Reply:
TBH, maybe it's only personal taste, but I don't really like to specify `auth_` properties in the code. For developers it's totally OK, but for administrators they are just hidden.

In any case, if they are in the code, IMHO they should ALSO be in the ConfigTemplate.cfg.

Reply:
Yes, I agree. Added to ConfigTemplate.cfg.

One thing, it seems to me that it would be useful to have default settings in the code, including that you can boot the service with a minimum configuration file, or without it at all.